src/platform_impl/web/event_loop/state.rs

```diff
@@ -6,6 +6,7 @@ use web_time::Instant;
 pub enum State {
     Init,
     WaitUntil {
+        #[allow(dead_code)]
         timeout: backend::Schedule,
```

Comment:
```suggestion
        _timeout: backend::Schedule,
```
This is stored here for its drop behavior.